orc8r/cloud/deploy/orc8r_deployer/docker/run_deployer

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+Print_Help()
+{
+   echo "./run_deployer runs the orc8r deployer container"
+   echo "orc8r deployer contains scripts which enable user to configure, run prechecks"
+   echo "install, upgrade, verify and cleanup an orc8r deployment"
+   echo
+   echo "Syntax: run_deployer [-d|-r|-b|-h]"
+   echo "options:"
+   echo "-h     Print this Help"
```

Comment:
Default to long-form of CLI flags -- easier to read, easier to convey meaning to users. Ref: https://magma.github.io/magma/docs/next/contributing/contribute_conventions#all-documentation